### PR TITLE
code sniffer でエラーが出る行を追加

### PR DESCRIPTION
**Pull request by clear-nishikubo**

### CODE REVIEW

File: app/Controller/PagesController.php [hunk 1]
```diff
@@ -36,6 +36,7 @@ class PagesController extends AppController {
  * @var array
  */
 	public $uses = array();
+	public $uses2 = array();
```
Comment (clear-nishikubo) [inline app/Controller/PagesController.php]: Spaces must be used to indent lines; tabs are not allowed

Comment (clear-nishikubo) [inline app/Controller/PagesController.php]: Spaces must be used to indent lines; tabs are not allowed

Comment (clear-nishikubo) [inline app/Controller/PagesController.php]: Spaces must be used to indent lines; tabs are not allowed

Comment (clear-nishikubo) [inline app/Controller/PagesController.php]: Spaces must be used to indent lines; tabs are not allowed

Comment (clear-nishikubo) [inline app/Controller/PagesController.php]: Spaces must be used to indent lines; tabs are not allowed

Comment (clear-nishikubo) [inline app/Controller/PagesController.php]: Spaces must be used to indent lines; tabs are not allowed

File: app/Controller/PagesController.php [hunk 2]
```diff
@@ -36,6 +36,8 @@ class PagesController extends AppController {
...
+	public $uses3 = array();
```
Comment (clear-nishikubo) [inline app/Controller/PagesController.php]: Spaces must be used to indent lines; tabs are not allowed

File: app/Controller/PagesController.php (hunk 1 above)
Comment (sumyapp) [inline app/Controller/PagesController.php]: Spaces must be used to indent lines; tabs are not allowed

File: app/Controller/PagesController.php (hunk 2 above)
Comment (sumyapp) [inline app/Controller/PagesController.php]: Spaces must be used to indent lines; tabs are not allowed

File: app/Controller/PagesController.php [hunk 3]
```diff
@@ -36,6 +36,8 @@ class PagesController extends AppController {
  * @var array
  */
 	public $uses = array();
+	public $uses2 = array();
```
Comment (sumyapp) [inline app/Controller/PagesController.php]: Spaces must be used to indent lines; tabs are not allowed

File: app/Controller/PagesController.php (hunk 2 above)
Comment (sumyapp) [inline app/Controller/PagesController.php]: Spaces must be used to indent lines; tabs are not allowed

File: app/Controller/PagesController.php (hunk 1 above)
Comment (clear-nishikubo) [inline app/Controller/PagesController.php]: Spaces must be used to indent lines; tabs are not allowed

Comment (clear-nishikubo) [inline app/Controller/PagesController.php]: [development] [code_sniffer]

Comment (clear-nishikubo) [inline app/Controller/PagesController.php]: [development] [code_sniffer] Spaces must be used to indent lines; tabs are not allowed

**Pull request closed without merging**